apps/bfd-model/bfd-model-rda/src/main/java/gov/cms/bfd/model/rda/PreAdjFissProcCodePK.java

```diff
@@ -0,0 +1,16 @@
+package gov.cms.bfd.model.rda;
+
+import java.io.Serializable;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+/* PK class for the PreAdjFissProcCodes table */
+public class PreAdjFissProcCodePK implements Serializable {
```

Comment:
Having this in its own top level class is fine but what do you think about making this a static inner class named PK inside of the class it's a key for?